src/datasets/arrow_dataset.py

```diff
@@ -461,6 +461,41 @@ def from_csv(
             path_or_paths, split=split, features=features, cache_dir=cache_dir, keep_in_memory=keep_in_memory, **kwargs
         ).read()
 
+    @staticmethod
+    def from_json(
+        path_or_paths: Union[PathLike, List[PathLike]],
+        split: Optional[NamedSplit] = None,
+        features: Optional[Features] = None,
+        cache_dir: str = None,
+        keep_in_memory: bool = False,
+        field: Optional[str] = None,
+        **kwargs,
+    ):
+        """Create Dataset from JSON (Lines) file(s).
```

Comment:
```suggestion
        """Create Dataset from JSON or JSON Lines file(s).
```
(nit) this is clearer this way imo

Reply:
Done